nbgrader/html/templates/student_submissions.tpl

```diff
@@ -27,9 +27,9 @@
   {%- for submission in submissions -%}
   <tr>
     <td>
-      <a href="/submissions/{{ submission.id }}">
+      {%- if notebook_server_exists -%}<a href="/submissions/{{ submission.id }}">{%- endif -%}
```

Comment:
Ah I was confused.  Where I put the abort(503) is wrong.  I see what's going on now...  Another commit on it's way!